addons/mail/models/ir_http.py

```diff
@@ -3,18 +3,19 @@
 import odoo
 from odoo import models
 from odoo.http import request
-from odoo.addons.mail.controllers.webclient import WebclientController
 from odoo.addons.mail.tools.discuss import Store
 
 
 class IrHttp(models.AbstractModel):
     _inherit = 'ir.http'
 
-    def lazy_session_info(self):
-        res = super().lazy_session_info()
-        res["store_data"] = WebclientController._process_request(
-            fetch_params=["failures", "systray_get_activities", "init_messaging"],
```

Comment:
Note: not all bundles need all the params, clients should be responsible for what they need to fetch.